geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -388,9 +378,21 @@ public void close(BucketRegion bucketRegion) {
     if (logger.isDebugEnabled()) {
       logger.debug("Clearing entries for {} rvv={}", _getOwner(), rvv);
     }
-    LocalRegion lr = _getOwner();
+    final LocalRegion lr = _getOwner();
     RegionVersionVector localRvv = lr.getVersionVector();
-    incClearCount(lr);
+
+    boolean isBucketRegion = false;
+
+    if (!(lr instanceof HARegion)) {
+      CachePerfStats stats = lr.getCachePerfStats();
```

Comment:
stats looks unused so it should be removed

Reply:
it is used right below this comment.

Reply:
I have pruned out the stats need after re-reviewing that code.